### ISSUE TITLE
Change continuous bar legend to horizontal bar legend

### ISSUE THREAD

**Issue by XavierCLL** [NCC-CNC/wheretowork#223]
Hi folk!

This could be a minimal change proposal (or not essential) but it can help the tool.  When there are many layers, the current design for the continuous bar legend uses a lot of vertical space:

![image](https://user-images.githubusercontent.com/11801453/182755094-ea21190f-6864-4529-91bc-ce01b505f3d9.png)

If we change it to the horizontal legend we reduce unused space, easier to read and could improve usability for users, mockup:

![Screenshot_20220803_200917-1](https://user-images.githubusercontent.com/11801453/182755700-8514b8cb-fb98-432e-95d9-bf0c0b5e2208.png)

**Comment by jeffreyhanson** [comment]
Yeah - awesome idea!

**Comment by DanWismer** [comment]
I prototyped a horizontal continuous legend. I went with changing the linear-gradient to go left to right. Then positioning everything else using a series of flex boxes. I just need to figure out if I can inject the value `to left` in the `background-image: linear-gradient` property in the code base.
![image](https://user-images.githubusercontent.com/29556279/182905611-7cffdb23-fccf-48b2-96bb-6be27e2729ea.png)


[continous-legend-prototype.zip](https://github.com/NCC-CNC/wheretowork/files/9261636/continous-legend-prototype.zip)
